src/patternfly/components/Compass/compass.scss

```diff
@@ -92,6 +94,24 @@
   min-height: 0;
 }
 
+.#{$compass}__main-header {
+  & .#{$compass}__panel {
+    display: flex;
+    flex-wrap: wrap;
+    gap: var(--#{$compass}__main-header--RowGap) var(--#{$compass}__main-header--ColumnGap);
+    align-items: center;
```

Comment:
If they don't wrap by default, we'd want to take this off and leave things top-aligned or baseline aligned or something like that, so that if the title text wraps, it wraps like the card 

<img width="550" height="240" alt="Image" src="https://github.com/user-attachments/assets/f428983d-3111-45ef-a8bc-9ad59bf95b0d" />